### PR TITLE
Add demo showing how to edit nodes in TreeVirtual

### PR DESCRIPTION
**Pull request by rad-pat**
A simple demo to show how to allow the editing of nodes in the TreeVirtual.

### CODE REVIEW

File: source/class/qxl/demobrowser/demo/treevirtual/TreeVirtual_NodeEdit.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                       {
+                         var data = e.getData();
+                         alert(`dataEdited from ${data.oldValue.label} to ${data.value.label}`);
+                       });
+
+      var instructionGroup = new qx.ui.groupbox.GroupBox("Instructions");
+      instructionGroup.setLayout(new qx.ui.layout.VBox(2));
+      hBox.add(instructionGroup);
+
+      var filterLabel = new qx.ui.basic.Label("Double-click any node to being the editing");
```
Comment (derrell) [inline source/class/qxl/demobrowser/demo/treevirtual/TreeVirtual_NodeEdit.js]: s/being/begin/

**Review by derrell — CHANGES_REQUESTED** [review]
Great to have a demo of this. Thanks.

**Review by cboulanger — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Comment by cboulanger** [comment]
Merged, thank you @rad-pat !

**Comment by hkollmann** [comment]
@rad-pat: It's not working: https://qooxdoo.org/qxl.demobrowser/#treevirtual~TreeVirtual_NodeEdit.html. 
Any idea?

**Comment by hkollmann** [comment]
Get's SCRIPT438: SCRIPT438: Object doesn't support property or method 'setAllowNodeEdit' in browser log.

**Comment by rad-pat** [comment]
Yep, I had to use latest master of qooxdoo when I built it locally, updating the compile.json to point to custom Qooxdoo - the latest version of compiler must not include the changes made.

**Comment by rad-pat** [comment]
Actually, thinking about it further, that was not the error that I got when building locally. I got the error whereby the event old and newValue were the same, which is fixed in the latest master of qooxdoo.  This would suggest a version of the compiler that has an even older version of qooxdoo library within it. Either way, with latest master of qooxdoo, the demo will work.

**Pull request merged**